cibuildwheel/util.py

```diff
@@ -121,7 +123,42 @@ def __repr__(self):
         return f'{self.__class__.__name__}{self.base_file_path!r})'
 
 
+class Architecture(str, Enum):
```

Comment:
Subclassing `str` is to satisfy `mypy`?

Reply:
No, this makes this a "string enum". That is, `Architecture.X` _is_ the value string X. This has a few odd repercussions (string methods are no longer valid Enum names), but makes the enum a bit more natural as it also works naturally as a string. `Architecture.X == "X"` is true, for example.

Reply:
This is required for cross-library enums, which is where I've used it before.

Reply:
Ah, cool! I didn't know about that; thanks!